<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reset</class>
 <widget class="QMainWindow" name="Reset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="AnimatedStackedWidget" name="pages">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Reset Device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>How do you want to reset this device?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCommandLinkButton" name="reinstallAll">
            <property name="text">
             <string>Reinstall Everything</string>
            </property>
            <property name="description">
             <string>Reinstalls all software on this device. No data will be lost. Outdated software will be updated to their latest version.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="resetUsers">
            <property name="text">
             <string>Reset Users</string>
            </property>
            <property name="description">
             <string>Delete all users, including all data from all users, but keep software intact</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="resetEverything">
            <property name="text">
             <string>Reset Everything</string>
            </property>
            <property name="description">
             <string>Erase everything on this device and start fresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Preparing to reset everything</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Here's what we're going to do</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>- All data in all user accounts will be erased
- All user installed software will be removed
- Data on any other connected devices will not be removed</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="lowerPane">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="iconLabel">
           <property name="text">
            <string>ts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="osNameLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">theShell OS</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset theme="go-previous"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>Next</string>
           </property>
           <property name="icon">
            <iconset theme="go-next">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScallop">
    <property name="title">
     <string>Syste&amp;m</string>
    </property>
    <addaction name="actionReboot_System"/>
    <addaction name="actionPower_Off"/>
   </widget>
   <addaction name="menuScallop"/>
  </widget>
  <action name="actionReboot_System">
   <property name="icon">
    <iconset theme="system-reboot"/>
   </property>
   <property name="text">
    <string>Reboot</string>
   </property>
  </action>
  <action name="actionPower_Off">
   <property name="icon">
    <iconset theme="system-shutdown"/>
   </property>
   <property name="text">
    <string>Power Off</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
